app/controllers/api/v1/api_controller.rb

```diff
@@ -6,7 +6,8 @@ def pundit_user
     end
 
     def current_organization
-      @current_organization ||= current_user.organizations.find(params[:organization_id])
+      @current_organization ||= current_user.organizations.find_by_id(params[:organization_id]) if params[:organization_id].present?
```

Comment:
@dissident it should not works this way. `current_organization` should depend only on provided parameter